styles/style.css

```
@@ -3,8 +3,109 @@ Colors to use:
 dark blue: #06365f
 light blue: #1c5380
 */
-
+*{
+  margin: 0;
+  padding: 0;
+}
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  
+}
+main {
+  display: flex;
+  padding-top: 50px;
+  padding-left: 200px;
+  padding-right: 200px;
+  background-color: #1c5380;
+  height: 220px;
+  background-repeat: no-repeat;
```

```suggestion
  display: flex;
  padding-top: 50px;
  padding-left: 200px;
  padding-right: 200px;
  height: 220px;
  background-image: url(../images/bgtile.png);
  background-repeat: repeat;
```
Podrías hacer uso de la imagen dada ( bgtile.png),otorgándole al fondo el  efecto  observado en el screenshot.